typescript/sdk/src/consts/multisigIsm.ts

```diff
@@ -4,7 +4,7 @@ import { ChainMap } from '../types';
 export const defaultMultisigIsmConfigs: ChainMap<MultisigIsmConfig> = {
   // ----------------- Mainnets -----------------
   celo: {
-    type: ModuleType.MULTISIG,
+    type: ModuleType.MERKLE_ROOT_MULTISIG,
```

Comment:
This should be legacy still, right? (and below)

Reply:
woops yes